### DISCUSSION TITLE
6CB1584C-DB4E-2D6B-19A5-A87179BD4A0A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#48] [General]
# 6CB1584C-DB4E-2D6B-19A5-A87179BD4A0A

Tianwen-1 is a space probe of the China National Space Administration (CNSA) destined for the planet Mars. It launched on July 23, 2020 and entered Mars orbit on February 10, 2021. Tianwen-1 also carried the Zhurong rover, which landed on the Martian surface on May 14, 2021. The Tianwen-1 orbiter and the rover Zhurong remain operating in 2022.
NOTE: This addon will only show Tianwen-1 on its Earth to Mars cruise trajectory. The probe's trajectory around Mars has not yet been added. Also note that this addon does not add the Zhurong rover.

https://celestia.mobi/addon?item=6CB1584C-DB4E-2D6B-19A5-A87179BD4A0A&lang=zh_CN

**Comment by 13940951572** [comment]
希望能有祝融号火星车！？

**Reply by DaveBowman2001** [reply]
There will be in the future. Hopefully an orbital phase to Tianwen-1 itself too, but complete trajectory data beyond February 2021 unfortunately isn't publicly available

**Reply by DaveBowman2001** [reply]
Updated the original addon on 2024-Jun-6, this should put Tianwen-1 into a permanent orbit around Mars. Zhurong is not yet included though

![Celestia2024-06-0614-22-48-ezgif com-video-to-gif-converter](https://github.com/celestiamobile/comments/assets/95486841/36435894-3b3b-4323-bb38-c7baa7813f00)
Here's a timelapse of Tianwen-1's first orbits around Mars in 2021
